src/Microsoft.FeatureManagement.AspNetCore/TagHelpers/FeatureTagHelper.cs

```diff
@@ -30,13 +31,19 @@ public class FeatureTagHelper : TagHelper
         /// </summary>
         public bool Negate { get; set; }
 
+        /// <summary>
+        /// A variant name, or comma separated list of variant names. If any of specified variants is assigned, the content should be rendered.
+        /// If variant is specified, <see cref="Negate"/> and <see cref="Requirement"/> will be ignored. Besides, <see cref="Name"/> should be exactly one feature name.
```

Comment:
what's the reason of ignoring `Negate`?

Reply:
I forget to update the comment. `Negate` is supported.